scripts/common.py

```diff
@@ -642,21 +640,6 @@ def get_personal_access_token() -> str:
     return personal_access_token
 
 
-def convert_to_posixpath(file_path: str) -> str:
```

Comment:
Note that convert_to_posixpath is a no-op for non-Windows systems, which is why it's been deleted from most of the code.